guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>Once the best performing network is identified, it can be further fine tuned using the <code>ImageryModel</code> class. This class supports methods that can be used to load, fine-tune, and save the model for further use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>